plugins/woocommerce/client/admin/client/homescreen/activity-panel/orders/index.js

```diff
@@ -313,9 +295,35 @@ function OrdersPanel( { unreadOrdersCount, orderStatuses } ) {
 			isRequesting: isRequestingActionable,
 			orderStatuses,
 			customerItems: customers,
+			currencySymbols: getCurrencySymbols() || [],
 		};
 	} );
 
+	const getFormattedOrderTotal = ( total, orderCurrencyCode ) => {
+		if ( ! orderCurrencyCode ) {
+			return null;
+		}
+
+		// If the order currency is the same as the store currency, we show the formatted amount.
+		if ( storeCurrency && storeCurrency.code === orderCurrencyCode ) {
+			return currencyContext.formatAmount( total );
+		}
+
+		const symbol = currencySymbols[ orderCurrencyCode ] ?? null;
+
+		if ( ! symbol ) {
+			// This should never happen, but if it does, we'll just show the currency code.
+			return `${ orderCurrencyCode }${ total }`;
+		}
+
+		// If the order currency is different from the store currency, we show the currency code and amount in the order currency.
+		return CurrencyFactory( {
+			...storeCurrency,
+			symbol: decodeEntities( symbol ),
```

Comment:
I understand that this is just the existing code, but I think it would be a good improvement to include the full currency settings here instead of only the symbol - for example if the order currency is set to USD and the store config is set to JPY, the currency symbol appears on the wrong side:

<img width="527" alt="image" src="https://github.com/user-attachments/assets/2f862cf9-e04b-491d-8ac9-b18d2b179d6f" />

Reply:
@rjchow That's a good find. Fix it in [Support include_extra. When include_extra is set, the endpoint re…](https://github.com/woocommerce/woocommerce/pull/55464/commits/332baa8e4585561a955d0a554701102cba65edce) and [Rename getCurrencySymbols to getCurrencies](https://github.com/woocommerce/woocommerce/pull/55464/commits/247d34fbe895298c33f969d808fb25346832a575)